mirrord/config/src/feature.rs

```diff
@@ -85,7 +85,7 @@ pub struct FeatureConfig {
     /// ## feature.copy_target {#feature-copy_target}
     ///
     /// Creates a new copy of the target. mirrord will use this copy instead of the original target
-    /// (e.g. intercept network traffic). This feature requires a [mirrord operator](https://mirrord.dev/docs/overview/teams/).
+    /// (e.g. intercept network traffic). This feature requires a [mirrord operator](https://mirrord.dev/docs/overview/teams/?utm_source=copytarget).
```

Comment:
do we actually want those query parameters in the website? not only in links that show in the CLI/IDE?

Reply:
maybe @eyalb181 has an opinion?

Reply:
I didn't think about it, but sure, why not. Would be interesting to learn if the docs are what leads people to mfT.